buffer-memory-segment/pom.xml

```diff
@@ -65,7 +64,14 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>${surefire.version}</version>
+        <configuration>
+          <argLine>-Dio.netty5.buffer.MemoryManager=MemorySegment</argLine>
+          <dependenciesToScan>
+            <dependency>io.netty:netty5-buffer</dependency>
+            <dependency>io.netty:netty5-handler</dependency>
+            <dependency>io.netty:netty5-codec-http</dependency>
```

Comment:
These 3 dependencies are added below as test dependencies, not sure if tests of the http and handler module should actually be executed as well (at least for the http module I can't see another reason for adding it). However, if that is the case, I do think that the test-jar type is missing on the last 2. Is this correct or should only the tests of the buffer module be executed on this module as well?

Reply:
I also don't see a reason the http module needs to be there. If we remove `EchoIT` then only the buffer module tests are needed, I think.